docs/gh-actions/gh-update.md

```diff
@@ -130,7 +140,7 @@ Option 2: If you chose to leave [TestFlight Automatic Updates](../gh-actions/gh-
 
 In the Loop app, once installed on your phone, tap on Settings -> Support -> Issue Report. The graphic below shows an example of the Build Details included in the report.
```

Comment:
To prevent automatic translation, I suggest enclosing them in backticks `` `Settings -> Support -> Issue Report` ``.